debugging-modules/debugging-modules.webidl

```diff
@@ -0,0 +1,224 @@
+typedef unsigned long WasmBreakpointId;
+typedef unsigned long WasmCodeOffset;
```

Comment:
I believe `unsigned long` is 32-bits in WebIDL <https://www.w3.org/TR/WebIDL-1/#idl-unsigned-long>. Is this specific to Wasm32?